File: ar-app-android-project/app/src/main/java/com/example/robotkarolar/karollogic_ben/instructions/statements/LeftTurn.java
package com.example.robotkarolar.karollogic_ben.instructions.statements;

import com.example.robotkarolar.karollogic_ben.instructions.Instruction;
import com.example.robotkarolar.karollogic_ben.instructions.visitors.InstructionVisitor;

public class LeftTurn extends Instruction {
    @Override
    public void accept(InstructionVisitor visitor) {
        visitor.accept(this);
    }
}

File: ar-app-android-project/app/src/main/java/com/example/robotkarolar/karollogic/instructions/statements/LeftTurn.kt
package com.example.robotkarolar.karollogic.instructions.statements

import com.example.robotkarolar.karollogic.instructions.Instruction
import com.example.robotkarolar.karollogic.instructions.visitors.InstructionVisitor
import kotlinx.parcelize.Parcelize

@Parcelize
class LeftTurn : Instruction() {
    override fun accept(visitor: InstructionVisitor?) {
        visitor!!.accept(this)
    }
}